cpp/unreal_plugins/SimulationController/Source/SimulationController/SimulationController.cpp

```diff
@@ -36,27 +36,21 @@
 #include "SimulationController/RpcServer.h"
 #include "SimulationController/SphereAgent.h"
 #include "SimulationController/Task.h"
+#include "SimulationController/UrdfBotAgent.h"
 #include "SimulationController/Visualizer.h"
 
 // Different possible frame states for thread synchronization
-enum class FrameState
```

Comment:
I like the old style for enums because it's more consistent with our other enums.

Reply:
Can you please revert this enum change for consistency with our existing code.

Reply:
Can you please revert this enum change so it's consistent with every other enum in our codebase: UrdfGeometryType, UrdfJointType, UrdfJointControlType, CalibrationType, DataType.